data/core/units/elves/Hunter.cfg

```diff
@@ -0,0 +1,212 @@
+#textdomain wesnoth-units
+
+[unit_type]
+    id=Elvish Hunter
+    name= _ "Elvish Hunter"
+    race=elf
+    gender=male,female
+    image="units/elves-wood/hunter.png"
+    profile="portraits/elves/archer.webp"
+    hitpoints=29
+    movement_type=woodland
+    movement=6
+    cost=16
+    experience=42
+    level=1
+    alignment=neutral
+    advances_to=Elvish Trapper
+    usage=mixed fighter
+    [movement_costs]
+        cave=2
+    [/movement_costs]
+    # These guys have a better defense in swamp water and sands
+    [defense]
+        swamp_water=60
+        cave=50
+        mountains=30
+        hills=40
+        sand=60
+    [/defense]
+    description=_"Elvish hunters are specialized in trapping unwary opponents from a safe distance to ease the task of the swordsmen."
```

Comment:
For a mainline unit, I'd expect a longer text. Maybe something about not all groups having shamans, or about the trappers having more movement than the shamans, maybe that the hunters aren't too bad in melee.

Probably should be something more gender-neutral than "swordsmen".

Reply:
I'm not opposed to making it gender-neutral, but it's probably mainly referring to the fighter line which is mechanically all-male.

Reply:
If this is still a problem, I can write one. You'd just have to tell me what it needs to say.  (If the unit is even going to be added)